config/config.go

```diff
@@ -1,6 +1,7 @@
 package config
 
 import (
+	kustomizev1beta2 "github.com/fluxcd/kustomize-controller/api/v1beta2"
```

Comment:
Just a note, I know `kustomize.toolkit.fluxcd.io/v1` already exists for a while but just do double check, in our MC's we only support `v1beta2` right now?

Reply:
good point!
I checked gaia and this is what we have:

```
$ kubectl get crd kustomizations.kustomize.toolkit.fluxcd.io -o yaml | yq '.spec.versions[]| .name'
v1beta1
v1beta2
```

but the real reason I am sticking to the old version is if I use the latest version we get in trouble with dependencies and I didn't want to go down that rabbit hole for now.